test/built-ins/Temporal/Instant/prototype/add/branding.js

```diff
@@ -11,12 +11,14 @@ const add = Temporal.Instant.prototype.add;
 
 assert.sameValue(typeof add, "function");
 
-assert.throws(TypeError, () => add.call(undefined), "undefined");
-assert.throws(TypeError, () => add.call(null), "null");
-assert.throws(TypeError, () => add.call(true), "true");
-assert.throws(TypeError, () => add.call(""), "empty string");
-assert.throws(TypeError, () => add.call(Symbol()), "symbol");
-assert.throws(TypeError, () => add.call(1), "1");
-assert.throws(TypeError, () => add.call({}), "plain object");
-assert.throws(TypeError, () => add.call(Temporal.Instant), "Temporal.Instant");
-assert.throws(TypeError, () => add.call(Temporal.Instant.prototype), "Temporal.Instant.prototype");
+const arg = new Temporal.Duration(0, 0, 0, 0, 5);
+
+assert.throws(TypeError, () => add.call(undefined, arg), "undefined");
+assert.throws(TypeError, () => add.call(null, arg), "null");
+assert.throws(TypeError, () => add.call(true, arg), "true");
+assert.throws(TypeError, () => add.call("", arg), "empty string");
+assert.throws(TypeError, () => add.call(Symbol(), arg), "symbol");
+assert.throws(TypeError, () => add.call(1, arg), "1");
+assert.throws(TypeError, () => add.call({}, arg), "plain object");
+assert.throws(TypeError, () => add.call(Temporal.Instant, arg), "Temporal.Instant");
+assert.throws(TypeError, () => add.call(Temporal.Instant.prototype, arg), "Temporal.Instant.prototype");
```

Comment:
Nice! I see that this fixes the ambiguity between step 2 and step 3 😄

Reply:
Yeah, I figured even though #3224 is still a problem in other branding tests (which I will tackle separately), Frank's original test nicely avoided that problem, so best to preserve it.